tests/int/pactGenerating.test.ts

```diff
@@ -25,6 +24,16 @@ describe('createPacts', () => {
         });
     });
 
+    test('pact-matcher', () => {
```

Comment:
Is it purposeful or some test residues?

Reply:
Actually after consideration - I reverted this change in this PR - lets do it all at once in other refactor task - but the idea stays the same.